src/components/MovieDetails/index.tsx

```diff
@@ -86,6 +90,8 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
   const intl = useIntl();
   const { locale } = useLocale();
   const [showManager, setShowManager] = useState(false);
+  const minStudios = 3;
+  const [studiosToShow, setStudiosToShow] = useState(minStudios);
```

Comment:
Perhaps this should be a boolean?  Since it is only ever one of two values (`minStudios` or the number of elements in the array).  Would then be able to eliminate the `studiosToShow === minStudios` comparisons below.